src/components/XPubTable/DataTable.tsx

```diff
@@ -32,6 +40,18 @@ export function DataTable<TData, TValue>({ columns, data }: DataTableProps<TData
     },
   });
 
+  const renderInfo = (obj: any) =>
```

Comment:
Is there a possibility to get rid of this `any` type and provide some more "specific" type?

Reply:
Will have a look on it.

Reply:
Added types